front/lock.form.php

```diff
@@ -39,19 +39,30 @@
 if (isset($_POST['itemtype'])) {
 
    $itemtype    = $_POST['itemtype'];
+   /* @var Computer $source_item */
```

Comment:
What's the purpose of that?

Reply:
Code assist in IDE.

Reply:
But it should not have been pushed here.